app/js/actions/patientAction.js

```diff
@@ -0,0 +1,48 @@
+import actionTypes from "./actionTypes";
+
+const getPatientRecord = patientUuid => ({
```

Comment:
Per our conventions, we can just call this "gePatient", we don't need to call it "getPatientRecord"

Same with "getPatientRecordSucceeded" and "getPatientRecordFailed"